### ISSUE TITLE
Ensure all interactive functionality is operable with the keyboard

### ISSUE THREAD

**Issue by blnkt** [lsst-epo/rubin-obs-client#61]
## Description
All interactive functionality needs to be operable with the keyboard. When interactive elements can not be navigated to or operated with the keyboard users who rely on the keyboard such as users of assistive technology may not be able to perform that function. This best practice addresses keyboard access to hoverable as well as expand/collapse content as well. Specific keystroke timings may not be relied upon as precise timing of keystrokes may be difficult or impossible for some users. There is an exception when the function relies on the path of the users input and not the endpoint

### Header

- [x] Language toggle button should have keyboard support.  Recommend changing to button element that communicates its state.  Like [the mute button found in this w3 example](https://www.w3.org/TR/wai-aria-practices-1.1/examples/button/button.html).

![0893a0c8e65de389e11ed02e1876fa0a967762ba-1641468432](https://user-images.githubusercontent.com/8166060/152620820-5ca53e54-781b-4ec8-9326-3afc3e98c303.png)
